spikeinterface/extractors/neoextractors/mearec.py

```diff
@@ -27,6 +27,9 @@ def __init__(self, file_path, all_annotations=False):
         probe = pi.read_mearec(file_path)
         self.set_probe(probe, in_place=True)
         self.annotate(is_filtered=True)
+        
+        if hasattr(self.neo_reader._recgen, "gain_to_uV"):
+            self.set_channel_gains(self.neo_reader._recgen.gain_to_uV)
```

Comment:
Normally mearec files are already scaled! So no need to add this gain.
When format is float32 in micro volt we do not set the gain.
@alejoe91 is this true ? gain_to_uV was added for what in mearec ?